templates/quickstart-jira-dc.template.yaml

```diff
@@ -12,6 +12,7 @@ Metadata:
       - Label:
           default: Cluster nodes
         Parameters:
+          - JiraClusterNodeRole
```

Comment:
I feel like this should be at the bottom of the `Cluster nodes` group because it is not a very important parameter.

Reply:
Done